code/game/objects/items/nanoforgedkatana.dm

```diff
@@ -0,0 +1,244 @@
+/obj/item/nanoforgedkatana
+	name = "Nanoforged Katana"
+	desc = "Glorious space nippon steel, folded a million times, producing the finest blade known to mankind. \
+			This blade seems to transmit knowledge of how to use its powers directly to its owner."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "nanoforgedkatana"
+	item_state = "nanoforgedkatana"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	flags_1 = CONDUCT_1
+	obj_flags = UNIQUE_RENAME
+	w_class = WEIGHT_CLASS_BULKY
+	force = 30
+	throwforce = 70
+	throw_speed = 3
+	throw_range = 7
+	sharpness = IS_SHARP_ACCURATE
+	embedding = list( "pain_mult" = 0, "jostle_pain_mult" = 0, "embed_chance" = 300, "armour_block" = 100, "fall_chance" = 0, "ignore_throwspeed_threshold" = TRUE)
+	// Last ditch effort to kill someone. Obviously, terrible idea unless the situation is desperate, as it WILL embed and WILL need surgery to remove. No extra embed damage tho, so as to not make it op.
+	hitsound = 'sound/weapons/anime_slash.ogg'
+	armour_penetration = 75
+	block_level = 2
```

Comment:
this is bad and infringes on desword's niche

Reply:
The armor penetration with lower damage ?
desword damage is 34 with 35 armor pen.
katana is 30 with 75 armor pen. think the damage should absolutely be lower, but the armor pen is the complete opposite. I meant this weapon as a sec/chap counter less than a "murder civilians lol"

Probably lowering the damage to like 27 ? so a 4 hit kill, lowered to 5 on sec, but can't be lowered any further?